<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>518</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Azure video to srt</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>400</y>
     <width>104</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>輸出 utf-8</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>430</y>
     <width>481</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>361</width>
     <height>221</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>10</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>微軟正黑體</family>
     <pointsize>12</pointsize>
     <weight>50</weight>
     <bold>false</bold>
    </font>
   </property>
   <property name="text">
    <string>Azure 影片轉 srt 字幕工具 v3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Transcribe">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>360</y>
     <width>104</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>連線轉換</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AzKey">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>50</y>
     <width>250</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>please_input_key</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_AzRegion">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>150</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>please_input_Region</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>52</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Key:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>83</y>
     <width>58</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Region:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>115</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>檔案路徑：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>157</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>輸出路徑：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FilePath">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_FileOutput">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>400</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_OpenFile">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>210</y>
     <width>104</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>開啓檔案</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_importKeyRegion">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>50</y>
     <width>131</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>匯入 Key / Region</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>190</y>
     <width>80</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>轉換訊息：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_TransSub">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>250</y>
     <width>104</width>
     <height>32</height>
    </rect>
   </property>
   <property name="text">
    <string>轉換字幕</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
